features/steps/steps.py

```diff
@@ -331,16 +406,25 @@ def step_impl(context, relationship_type, entity):
 
 @given('A file with {field} "{values}"')
 def step_impl(context, field, values):
-    values = list(map(str.lower, map(lambda s: s.strip('"'), values.split(' or '))))
+    values = strip_split(values, strp = '"', splt = ' or ')
     if field == "Model View Definition":
         conditional_lowercase = lambda s: s.lower() if s else None
         applicable = conditional_lowercase(get_mvd(context.model)) in values
     elif field == "Schema Identifier":
         applicable = context.model.schema.lower() in values
     else:
         raise NotImplementedError(f'A file with "{field}" is not implemented')
-
+ 
     context.applicable = getattr(context, 'applicable', True) and applicable
+    
+@given('Its values')
```

Comment:
And then you can just say

```suggestion
@given('Its values')
@given('Its values excluding {excluding}')
def step_impl(context, excluding=()):
```

And just pass that to ignore directly.